### PR TITLE
Home

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Added:home page
Note: its just a basic layout, i will add color and make it responsive later in the end
image
image
image
https://condescending-poincare-6dbeb0.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,98 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (yashpatel1797) [inline index.html:6]: add author tag

File: styles/home.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.footer-items > h4 {
+  margin: 0.5rem auto;
+  text-align: center;
+}
+
+.footer-icon-holder {
+  display: flex;
+  flex-direction: row;
```
Comment (yashpatel1797) [inline styles/home.css:79]: flex direction is by default row so no need to declare here.

**Review by yashpatel1797 — APPROVED** [review]
add few suggestion. Code looks good to merge.

File: styles/home.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+
+.category {
+  padding: 1.5rem 2rem;
+}
+
+.col-1,
+.col-2,
+.col-3 {
```
Comment (poojavishnoi) [inline styles/home.css:35]: You can use one class name like column.

**Review by poojavishnoi — APPROVED** [review]
Looks good to me

File: styles/home.css (hunk 3 above)
Comment (anandkumarcoder) [reply styles/home.css:35]: will do so

File: index.html (hunk 1 above)
Comment (anandkumarcoder) [reply index.html:6]: done

**Pull request merged**